filecoin-proofs/src/api/mod.rs

```diff
@@ -136,68 +136,69 @@ pub unsafe extern "C" fn generate_post(
 ///
 #[no_mangle]
 pub unsafe extern "C" fn verify_post(
-    _flattened_comm_rs_ptr: *const u8,
```

Comment:
I think this is inadvertently reverting the work from https://github.com/filecoin-project/rust-fil-proofs/pull/490.

Reply:
> I think this is inadvertently reverting the work from #490.

In fact I did want to replace that portion of #490. I want `generate_post` `verify_post` in `mod.rs` to call in to their equivalents in `internal.rs` so that I could test those functions in the `ffi/main.rs` test.